document/src/main/java/com/ritense/document/web/rest/impl/JsonSchemaDocumentSnapshotResource.java

```diff
@@ -39,6 +40,7 @@
 
 @RestController
 @RequestMapping(value = "/api", produces = APPLICATION_JSON_UTF8_VALUE)
+@ConditionalOnBean(DocumentSnapshotService.class)
```

Comment:
Is this technically a breaking change? If so, let's not forget to mention it in documentation

Reply:
No, because the bean cannot be created without the DocumentSnapshotService anyway. It would just crash.